src/main/java/seedu/address/model/event/Event.java

```diff
@@ -61,7 +58,8 @@ public boolean isSameEvent(Event otherEvent) {
 
         return otherEvent != null
                 && otherEvent.getEventName().equals(getEventName())
-                && otherEvent.getDate().equals(getDate());
+                && otherEvent.getDate().equals(getDate())
+                && otherEvent.getAttendees().equals(getAttendees());
```

Comment:
Maybe we can keep `isSameEvent` to just name and date. If we add attendees, wouldn't it be possible to have 2 events with the same name and date but different attendees?